CLOUDFRONT.md

```diff
@@ -58,29 +56,12 @@ php admin/cli/scheduled_task.php --execute='\tool_objectfs\task\generate_status_
 6. Access the _/admin/tool/objectfs/object_status.php_ page.
 7. Confirm, that all files have been moved to the external storage: _Marked as only in filedir_ and _Duplicated in filedir and external storage_ should be 0.
 
-### Create CloudFront distribution
-1. Navigate to [https://console.aws.amazon.com/cloudfront/v3/home?region=ap-southeast-2#/welcome].
-2. Click on _Create a CloudFront distribution_.
-3. Choose your Amazon S3 bucket from _Origin domain_ dropdown menu.
-4. _S3 bucket access_: Choose _Yes use OAI (bucket can restrict access to only CloudFront)_ and click _Create new OAI_.
-5. _S3 bucket access -> Bucket policy_: Choose _Yes, update the bucket policy_.
-6. _Viewer protocol policy_: Choose _Redirect HTTP to HTTPS_.
-7. _Allowed HTTP methods_: Choose _GET, HEAD, OPTIONS_ and tick _OPTIONS_ under _Cache HTTP methods_.
-8. _Restrict viewer access_: Choose _Yes -> Trusted signer -> Self_.
-9. _Cache key and origin requests_: Choose _Legacy cache settings_.
-10. _Legacy cache settings -> Headers_: Choose _Include the following headers_ and add _Origin_, _Access-Control-Request-Method_, _Access-Control-Request-Headers_ headers from the dropdown menu.
-11. _Legacy cache settings -> Query strings_: Choose _All_.
-12. Click _Create distribution_.
-13. Navigate to [https://console.aws.amazon.com/cloudfront/v3/home?region=ap-southeast-2#/distributions].
-14. Confirm, that _Status_ is _Enabled_ and _Last modified_ is changed from _Deploying_ to the date the distribution was created.
-15. Open your distribution.
-16. Write down _Distribution domain name_ (with https://).
-> Note: If you have already setup Moodle behind a CloudFront distribution, it is also possible to use that same CloudFront distribution to serve files from objectfs. In this scenario, a specific prefix in the URL path directs traffic to the S3 Bucket (moodle.domain/objectfs/ for example). To achieve that, use the key_prefix option to add a prefix on your Bucket, and configure a second Origin on your existing CloudFront distribution that points to your Bucket. Setup a Behavior that uses that new Origin with the same prefix as the one you used as key_prefix in your Bucket. Follow all other instructions.
+### Generate CloudFront keys
+ObjectFS can use a CloudFront key from either the local filesystem or an admin setting. If using an admin setting, the key may be generated outside of the Moodle environment.
 
-### Generate keys
 1. Make a directory _$CFG->dataroot . '/objectfs/'_.
 2. Make it readable and writable:
-```
+ ```
```

Comment:
Seems like an extra space in the beginning of the line

Reply:
Thanks, fixed.